### ISSUE TITLE
Philip's blog

### ISSUE THREAD

**Issue by p208p2002** [p208p2002/blog#25]
https://blog.philip-huang.tech/?page=blog_bart-model_nlp 

 

<document-info>
- tags:  bart-model nlp
- date: 2020/01/01
</document-info>

   Import

```python
! pip install transformers==4.5.1
```

```python
from transformers import AutoTokenizer, AutoModelForSeq2SeqLM
import torch
```

```python
tokenizer = AutoTokenizer.from_pretrained('facebook/bart-base')
model = AutoModelForSeq2SeqLM.from_pretrained('facebook/bart-base')
```

   Prepare data

```python
context = 'Harry Potter is a series of'
label = 'seven fantasy novels' + tokenizer.eos_token
`
